### ISSUE TITLE
Obelisk of Light animation and sound don't work like intended

### ISSUE THREAD

**Issue by ZxGanon** [DoGyAUT/cd#21]
Obelisk of Light Effect and Sound broken.

**Comment by GraionDilach** [comment]
Doesn't have a voxel.

**Comment by DoGyAUT** [comment]
It's because AttackCharge has been replaced with AttackCharges which handles a few things different.

**Comment by GraionDilach** [comment]
The legacy behaviour is still there as AttackTesla or something along.

**Comment by abcdefg30** [comment]
Yeah, `AttackTesla` and `WithTeslaChargeAnimation` / `WithTeslaChargeOverlay`.
